apps/filebrowser/src/filebrowser/templates/listdir.mako

```diff
@@ -105,6 +105,13 @@ ${ fb_components.menubar() }
                   <i class="fa fa-fw fa-hdd-o"></i> ${_('Set replication')}
                 </a>
               </li>
+              % if not is_trash_enabled:
+              <li>
+                <a href="javascript: void(0)" class="delete-link" title="${_('Delete forever')}" data-bind="visible: !inTrash() && selectedFiles().length > 0 && isCurrentDirSelected().length == 0, click: deleteSelected">
+                  <i class="fa fa-bolt"></i> ${_('Delete forever')}
```

Comment:
Need `fa-fw` (we can see the icon not aligned)?

Reply:
Done